sklearn/utils/tests/test_array_api.py

```diff
@@ -143,6 +146,32 @@ def test_array_api_wrapper_take():
         xp.take(xp.asarray([[[0]]]), xp.asarray([0]), axis=0)
 
 
+def test_array_api_wrapper_searchsorted():
+    """Test _ArrayAPIWrapper API for searchsorted."""
+    numpy_array_api = pytest.importorskip("numpy.array_api")
```

Comment:
This will need to be completely rewritten or even removed.

`numpy.array_api` went away and is now replaced by `array-api-strict` (for compliance testing purposes), but we should not assume that `array-api-strict` is backed by NumPy or attempt any performance optimization for this library. It's not meant to be used by end users.